text/0000-security-advisories.md

```diff
@@ -0,0 +1,259 @@
+- Feature Name: security_advisories
+- Start Date: 2016-08-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Crates.io should offer an API to release security advisories for crates. Cargo
+should offer a new command for the same purpose and warn about vulnerable crate
+versions during compilation.
+
+# Motivation
+[motivation]: #motivation
+
+When compared to other ecosystems such as Python's, Rust's broader community
+prefers many single-purpose crates over larger monoliths. This situation,
+together with the strongly encouraged practice of pinning MINOR versions of
+dependencies, slows down the propagation of critical security fixes.
```

Comment:
I feel this bit of motivation doesn't work because
 * pinning to specific versions is not the norm,
 * cargo makes the ultimate consumer track precise versions via lockfile, giving them more power and responsibility.

So applying the fix should normally not be a problem. Only if the intermediate crates actively prevent this by strict version pinning do you need to act differently (e.g. prevent publishing a crate, which makes it impossible to avoid vulnerable dependency versions).